app/code/Magebit/PageListWidget/view/frontend/templates/widget/page-list.phtml

```diff
@@ -0,0 +1,22 @@
+<?php
```

Comment:
Look into $escaper and what it does. 
All content must be escaped.
<?php echo can be shortened